Sources/Build/Buildable.swift

```diff
@@ -9,20 +9,43 @@
 */
 
 import PackageType
+import struct Utility.Path
 
 protocol Buildable {
     var targetName: String { get }
     var isTest: Bool { get }
 }
 
+extension CModule {
+    func workingDirectory(prefix: String) -> String {
```

Comment:
Can we get a documentation comment explaining what this function is?

Also, I think "workingDirectory" isn't the best name, something like "buildDirectory" would be closer to the intent.